egresocovid19/main.py

```diff
@@ -25,7 +29,21 @@ async def startup():
         database=client.egresocovid19,
         document_models=entities,  # type: ignore
     )
-
+    if len(await ProvinceEntity.find_all().to_list()) == 0:
```

Comment:
use `count` method of beanie.

![image](https://user-images.githubusercontent.com/36774373/133873462-e0f662b2-fb20-4d35-bc4e-55803c20e495.png)

https://roman-right.github.io/beanie/api-documentation/query/

`if not await ProvinceEntity.find_all().count():`